package/MDAnalysis/core/topologyattrs.py

```diff
@@ -790,13 +790,12 @@ def center_of_mass(group, compound='group', pbc=None):
         Parameters
         ----------
         compound : {'group', 'segments', 'residues'}, optional
-            If 'group', the center of mass of all atoms in the group will be
-            returned as a single position vector.
-            Else, the centers of mass of each segment or residue will be
-            returned as an array of position vectors, i.e. a 2d array.
-            Note that, in any case, *only* the positions and masses of atoms
-            *belonging to the group* will be taken into account.
-            [``'group'``]
+            If 'group', the center of mass of all atoms in the atomgroup will
+            be returned as a single position vector. Else, the centers of mass
+            of each segment or residue will be returned as an array of position
+            vectors, i.e. a 2d array. Note that, in any case, *only* the
+            positions of atoms *belonging to the atomgroup* will be
```

Comment:
Only atomgroups have this method as when I check it in an interpreter. So it didn't make sense to me to speak about general groups if they don't have this function.

Reply:
I have no problems calling `u.residues.center*()`. And `u.residues` is a `ResidueGroup` afaik.

Reply:
yeah I never know what that would sensibly return. I rather assume that `u.residues[0].center*` would work because it makes more sense to me. @richardjgowers can you commen?

Reply:
@kain88-de I think we tried to make `Residue.thing` only ever refer to the residue as a singular object, and `Residue.atoms.thing` work on the contents of the residue.  

It does come up often, maybe it's worth looking at what happens if we allow a `Residue` to sometimes act like an `AtomGroup`....